proposals/Proposal.md

```diff
@@ -0,0 +1,113 @@
+#﻿Improving reproducibility in science by adding provenance tracking to the EcoData Retriever.
+
+
+#Personal Details:-
+Name:- Prabh Simran Singh Baweja
+3rd Year, B.Tech in Computer Science and Engineering, IIIT Hyderabad, India.
+Email Id:- prabhsimransingh.baweja@gmail.com
+Website:- web.iiit.ac.in/~prabhsimransingh.baweja
+Phone Number:- +91-8790616675.

```

Comment:
You can remove you phone number, check a19315a. If you could squash your commits for this information not be under the version control will be great.